crates/apub/src/api/list_comments.rs

```diff
@@ -14,15 +14,18 @@ use lemmy_db_schema::{
   source::{comment::Comment, community::Community, local_site::LocalSite},
   traits::Crud,
 };
-use lemmy_db_views::comment_view::CommentQuery;
+use lemmy_db_views::{comment_view::CommentQuery, structs::LocalUserView};
 use lemmy_utils::error::{LemmyError, LemmyErrorExt, LemmyErrorType};
 
 #[tracing::instrument(skip(context))]
 pub async fn list_comments(
   data: Query<GetComments>,
   context: Data<LemmyContext>,
+  mut local_user_view: Option<LocalUserView>,
 ) -> Result<Json<GetCommentsResponse>, LemmyError> {
-  let local_user_view = local_user_view_from_jwt_opt(data.auth.as_ref(), &context).await;
+  if local_user_view.is_none() {
+    local_user_view = local_user_view_from_jwt_opt(data.auth.as_ref(), &context).await;
+  }
```

Comment:
This is necessary for all backwards compatibility in all api handlers. I will move the logic inside of the helper method so it wont create extra complexity.